app/routes/reminders/new.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
```

Comment:
@martensonbj We are using this for the route that our new-reminder component lives on. If I comment out this code, our route breaks.  Is there a better way to do this?

@mlimberg